subt/ros/robot/launch/x4.launch

```diff
@@ -140,8 +140,21 @@
     <remap from="odometry/filtered" to="/$(arg robot_name)/odom_fused"/>
   </node>
 
+  <node pkg="nodelet" type="nodelet" name="pcl_manager" args="manager" output="screen" />
+
+  <!-- Run a passthrough filter to remove drone propellers -->
+  <node pkg="nodelet" type="nodelet" name="passthrough" args="load pcl/PassThrough pcl_manager" output="screen">
+    <remap from="~input" to="/$(arg robot_name)/rgbd_camera/depth/points" />
+    <rosparam>
+      filter_field_name: z
+      filter_limit_min: 0.31
+      filter_limit_max: 100.0
+      filter_limit_negative: False
+    </rosparam>
+  </node>
+
   <node if="$(eval arg('robot_name').endswith('XM'))" ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
-    <remap from="cloud_in" to="/$(arg robot_name)/rgbd_camera/depth/points"/>
+    <remap from="cloud_in" to="/passthrough/output"/>
```

Comment:
Would you mind remapping `/passthrough/output` to some more descriptive topic name? It will be easier to understand with tools like `rostopic` or `rviz`.

Reply:
Any suggestion? For me this was like blind "copy and paste" from example so `output` is probably hardcoded, but `passthrough` node can be easily changed??

Reply:
Let's say you add `<remap from="/passthrough/output" to="/$(arg robot_name)/rgbd_camera/depth/points_cleaned" />` and use `/$(arg robot_name)/rgbd_camera/depth/points_cleaned` here?